source/Nuke.Common/Tools/SqlPackage/SqlPackage.json

```diff
@@ -0,0 +1,3145 @@
+                "name": "DisableAndReenableDdlTriggers",
+                "type": "bool",
+                "help": "Specifies whether Data Definition Language (DDL) triggers are disabled at the beginning of the publish process and re-enabled at the end of the publish action."
+              },
+              {
+                "name": "DisableIndexesForDataPhase",
+                "type": "bool",
+                "help": "Disable indexes before importing data into SQL Server."
+              },
+              {
+                "name": "DisableParallelismForEnablingIndexes",
+                "type": "bool",
+                "help": "Not using parallelism when rebuilding indexes while importing data into SQL Server."
+              },
+              {
+                "name": "DoNotAlterChangeDataCaptureObjects",
+                "type": "bool",
+                "help": "If true, Change Data Capture objects are not altered."
+              },
+              {
+                "name": "DoNotAlterReplicatedObjects",
+                "type": "bool",
+                "help": "Specifies whether objects that are replicated are identified during verification."
+              },
+              {
+                "name": "DoNotDropDatabaseWorkloadGroups",
+                "type": "bool",
+                "help": "When false, Database WorkloadGroups in the target database that are not defined in the source will be dropped during deployment."
+              },
+              {
+                "name": "DoNotDropObjectType",
+                "type": "string",
+                "help": "An object type that should not be dropped when DropObjectsNotInSource is true. Valid object type names are Aggregates, ApplicationRoles, Assemblies, AssemblyFiles, AsymmetricKeys, BrokerPriorities, Certificates, ColumnEncryptionKeys, ColumnMasterKeys, Contracts, DatabaseOptions, DatabaseRoles, DatabaseTriggers, Defaults, ExtendedProperties, ExternalDataSources, ExternalFileFormats, ExternalTables, Filegroups, Files, FileTables, FullTextCatalogs, FullTextStoplists, MessageTypes, PartitionFunctions, PartitionSchemes, Permissions, Queues, RemoteServiceBindings, RoleMembership, Rules, ScalarValuedFunctions, SearchPropertyLists, SecurityPolicies, Sequences, Services, Signatures, StoredProcedures, SymmetricKeys, Synonyms, Tables, TableValuedFunctions, UserDefinedDataTypes, UserDefinedTableTypes, ClrUserDefinedTypes, Users, Views, XmlSchemaCollections, Audits, Credentials, CryptographicProviders, DatabaseAuditSpecifications, DatabaseEncryptionKeys, DatabaseScopedCredentials, Endpoints, ErrorMessages, EventNotifications, EventSessions, LinkedServerLogins, LinkedServers, Logins, MasterKeys, Routes, ServerAuditSpecifications, ServerRoleMembership, ServerRoles, ServerTriggers, ExternalStreams, ExternalStreamingJobs, DatabaseWorkloadGroups, WorkloadClassifiers, ExternalLibraries, ExternalLanguages. This property may be specified multiple times to indicate multiple options."
```

Comment:
This (and others) should use `<ul>` and `<li>`.

Reply:
![image](https://github.com/nuke-build/nuke/assets/14993958/17af84b8-d637-40a6-8637-0bb5acd68ae3)
 
 I don't understand why? Looks like this is generating the required documentations  
![image](https://github.com/nuke-build/nuke/assets/14993958/c44023f4-e506-46cf-99c8-6782fb1aeff6)

Reply:
> Valid object type names are <enumeration>

Reply:
Do you mean using <ul> and <li> can make the valid values into strongly typed nuke string enumerations ?